### PR TITLE
#616 remove is_active in product

### PR DESCRIPTION
**Pull request by nguyenvanhadncntt**

### CODE REVIEW

File: product/src/main/java/com/yas/product/repository/ProductRepository.java [hunk 1]
```diff
@@ -53,7 +51,7 @@ Page<Product> getProductsWithFilter(@Param("productName") String productName,
             "AND (:endPrice IS NULL OR p.price <= :endPrice) " +
             "AND p.isVisibleIndividually = TRUE " +
             "AND p.isPublished = TRUE " +
-            "AND p.isActive = TRUE " +
+            "AND p.isPublished = TRUE " +
```
Comment (LEHOANGGLAM) [inline product/src/main/java/com/yas/product/repository/ProductRepository.java:54]: duplicated

**Review by LEHOANGGLAM — COMMENTED** [review]
remove duplicated row

File: product/src/main/java/com/yas/product/service/ProductService.java [hunk 2]
```diff
@@ -166,6 +166,7 @@ public ProductGetDetailVm createProduct(ProductPostVm productPostVm) {
                                 .sku(variation.sku())
                                 .gtin(variation.gtin())
                                 .price(variation.price())
+                                .isPublished(true)
```
Comment (khoahd7621) [inline product/src/main/java/com/yas/product/service/ProductService.java:169]: Don't hardcode here, it should be based on product parent's data.

**Review by khoahd7621 — CHANGES_REQUESTED** [review]
Reviewed, need a change.

**Review by khoahd7621 — APPROVED** [review]
Reviewed

**Pull request merged**